ngx-fudis/projects/ngx-fudis/src/lib/components/form/fieldset/fieldset.component.ts

```diff
@@ -182,6 +190,10 @@ export class FieldSetComponent
 
       this._resizeObserver.observe(this._fieldsetLegend.nativeElement);
     }
+
+    if (['fudis-radio-button-group', 'fudis-checkbox-group'].some((id) => this.id.includes(id))) {
```

Comment:
Change this id dependant check to the one where we just check if Fieldset has CheckboxGroup or RadioButtonGroup component as its parent.

- Add CheckboxGroup and RadioButtonGroup components to constructor with `@Host()` and `@Optional()` decorators.
- Change this if to check `this._parentCheckBoxGroup || this._parentRadioButtonGroup`

Reply:
This (of course) caused import cycles so we need to do it through ElementRef.